src/app/view/components/listing-data-table/listing-data-table.component.ts

```diff
@@ -62,10 +77,12 @@ export class ListingDataTableComponent extends BaseComponent {
      * @memberof ListingDataTableComponent
      */
     loadData(limit: number, offset: number, sortField?: string, sortOrder?: number, query?: string): void {
-        this.crudService.loadData(this.url, limit, offset, sortField,
-        sortOrder < 0 ? OrderDirection.DESC : OrderDirection.ASC, query)
-            .subscribe( response => {
-                this.items = response.items;
+        sortField = sortField || this.sortField;
+        sortOrder = sortOrder || (this.sortOrder === 'desc' ? -1 : 1);
```

Comment:
I didn't see this before but... here instead of this, you can pass the comparation that Freddy did in the calling of the method:

```OrderDirection.DESC : OrderDirection.ASC```